### ISSUE TITLE
Python library documentation shows calls but not responses for all sections besides database

### ISSUE THREAD

**Issue by ashrobertsdragon** [supabase/supabase#22070]
# Improve documentation

## Describe the problem

In the auth section, we are given a code sample like so:
`res = supabase.auth.sign_up({
  "email": 'example@email.com',
  "password": 'example-password',
})`

But in the database section, we are given the code sample for a call:
`response = supabase.table('countries').select("*").execute()`

and also the response:
`APIResponse(data=[{'id': 1, 'name': 'Afghanistan'},
                  {'id': 2, 'name': 'Albania'},
                  {'id': 3, 'name': 'Algeria'}],
            count=None)`

Instead of anything like this, the total information we are given for understanding the response to this call for signing up a user is this:

> - Confirm email determines if users need to confirm their email address after signing up.
>   - If Confirm email is enabled, a user is returned but session is null.
>   - If Confirm email is disabled, both a user and a session are returned.

## Describe the improvement

Sample responses to all of the methods shown in the documentation should be given

## Additional context
![Screenshot 2024-03-17 011034](https://github.com/supabase/supabase/assets/128648365/60ce1da6-b6a7-49e4-be49-c5ae6cd89644)

**Comment by Mrinank-Bhowmick** [comment]
Hey @ashrobertsdragon 
[supabase-py](https://github.com/supabase-community/supabase-py) changed codebase now the supabase.auth.sign_up() function is accepting dictionary and not individual parameters.

**Comment by ashrobertsdragon** [comment]
> Hey @ashrobertsdragon [supabase-py](https://github.com/supabase-community/supabase-py) changed codebase now the supabase.auth.sign_up() function is accepting dictionary and not individual parameters.

Right, the code sample I gave shows a dictionary being passed to sign_up()

**Comment by ashrobertsdragon** [comment]
I think it looks good, but I don't have any power to do anything

**Comment by Hallidayo** [comment]
Hi everyone, As a pull request has been merged that fixes the issue I'm going to close this one now.

**Comment by ashrobertsdragon** [comment]
I feel like everyone missed the point here, I used that single method as an example. But the entirety the rest of the auth methods still have no responses shown in the documentation, none of the storage methods have responses shown in the documentation. 
And of course, only about half of the auth methods are even documented at all

**Comment by Hallidayo** [comment]
Hi @ashrobertsdragon - ah sorry, I’ll get this re-opened

**Comment by silentworks** [comment]
The auth methods that have responses should be fixed now. The storage methods have a PR waiting to be merged that will fix this issue too. 

PR: https://github.com/supabase/supabase/pull/30105
